cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -52,15 +65,70 @@ def test_myservice_crud(appliance, setup_provider, context, order_catalog_item_i
+        vm_obj = myservice.launch_vm_console(catalog_item)
+        vm_console = vm_obj.vm_console
+        request.addfinalizer(vm_console.close_console_window)
+        request.addfinalizer(appliance.server.logout)
+        vm_ssh_client = ssh_client(vm_obj, console_template)
+        try:
+            assert vm_console.wait_for_connect(180), "VM Console did not reach 'connected' state"
+            user_count_before_login = vm_ssh_client.run_command("who --count", ensure_user=True)
+            assert vm_console.wait_for_text(text_to_find="login:", timeout=200), ("VM Console"
+                " didn't prompt for Login")
+            # Enter Username:
+            vm_console.send_keys("{}".format(console_vm_username))
+            assert vm_console.wait_for_text(text_to_find="Password", timeout=200), ("VM Console"
+            " didn't prompt for Password")
+            # Enter Password:
+            vm_console.send_keys("{}".format(console_vm_password))
+            logger.info("Wait to get the '$' prompt")
+            if catalog_item.provider.one_of(VMwareProvider):
+                vm_console.wait_for_text(text_to_find=catalog_item.provider.data
+                    .templates.get('console_template')['prompt_text'], timeout=200)
+            else:
+                # Since the RHV Console template needs to be updated, it does not have 'prompt_text'
+                # at this moment, once it is available, this time.sleep() call will be removed
+                # TODO: Remove time.sleep()
+                time.sleep(15)
+            user_count_after_login = vm_ssh_client.run_command("who --count", ensure_user=True)
+            # Number of users before login would be 0 and after login would be 180
+            # If below assertion would fail user_count_after_login is also 0, denoting login failed
+            assert (user_count_before_login.output.split('=')[-1].strip() <
+                    user_count_after_login.output.split('=')[-1].strip()), "Login Failed"
+            logger.info("Wait to get the '$' prompt")
+
+            vm_console.wait_for_text(text_to_find=catalog_item.provider.data.templates.
+                get('console_template')['prompt_text'], timeout=200)
+            # create file on system
+            vm_console.send_keys("touch blather")
+            wait_for(func=vm_ssh_client.run_command, func_args=["ls blather"],
+                func_kwargs={'ensure_user': True},
+                fail_condition=lambda result: result.rc != 0, delay=1, num_sec=10)
+            # if file was created in previous steps it will be removed here
+            # we will get instance of SSHResult
+            # Sometimes Openstack drops characters from word 'blather' hence try to remove
+            # file using partial file name. Known issue, being worked on.
+            command_result = vm_ssh_client.run_command("rm blather", ensure_user=True)
+            assert command_result
+
+        except:
```

Comment:
Required: One small thing here: specify an exception type here (as an option Exception can be used here)